### DISCUSSION TITLE
change color not working with scene-view

### DISCUSSION THREAD

**Discussion by dungnguyen234** [google/model-viewer#4229] [Q&A]
https://modelviewer.dev/examples/scenegraph/#changeColor
i try this  feature change color  but on android it load webXR , however i want to keep the effect when 3d model fall down to floor of scene-view , any chance to use this feature with scene-view mode ?

**Comment by elalish** [comment]
Unfortunately that isn't possible due to Scene Viewer redownloading the original GLB URL. Why do you want to use Scene Viewer instead of WebXR? I ask so we know how to improve our WebXR experience.

**Reply by dungnguyen234** [reply]
yes, in web XR don't have screenshot and record button , don't display number of percentage when scale , and don't have scan guidance at first start like scene viewer
